packages/core/src/definitions/styles.ts

```diff
@@ -1,7 +1,7 @@
 import { ComponentDefinitionVariable, ContainerStyleVariableName } from '@/types';
 
 export const builtInStyles: Partial<
-  Record<ContainerStyleVariableName, ComponentDefinitionVariable<'Text' | 'Boolean'>>
```

Comment:
Good catch getting all these instances of background images. I didn't think of that.